demo.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
```

Comment:
We've added demo puppeteer scripts to a bunch of PRs now. At some point we should start converting our demos into smoke tests.

Reply:
I agree we should start thinking about our approach to more comprehensive integration tests as we expand audit cases 👍  

At least in most of my cases though, that doesn't solve my problem since I need the CLI to support a `--fraggle-rock` mode, I just want to look through the report :) That's on the radar as well.